pkg/config/config.go

```diff
@@ -47,16 +47,31 @@ var c config
 var (
 	EnabledEBPFCgroupID = false
 
-	EstimatorModel        = "" // auto-select
-	EstimatorSelectFilter = "" // no filter
-	CoreUsageMetric       = "curr_cpu_cycles"
-	DRAMUsageMetric       = "curr_cache_miss"
-	UncoreUsageMetric     = ""                // no metric (evenly divided)
-	GeneralUsageMetric    = "curr_cpu_cycles" // for uncategorized energy; pkg - core - uncore
+	EstimatorModel        = getConfig("EstimatorModel", "")        // auto-select
+	EstimatorSelectFilter = getConfig("EstimatorSelectFilter", "") // no filter
+	CoreUsageMetric       = getConfig("CoreUsageMetric", "cpu_instr")
```

Comment:
The names of core, dram, and instruction metrics are defined in [bcc_attacher.go](https://github.com/sustainable-computing-io/kepler/blob/main/pkg/attacher/bcc_attacher.go#L51).

So I think the default values should be those of the [bcc_attacher.go](https://github.com/sustainable-computing-io/kepler/blob/main/pkg/attacher/bcc_attacher.go#L51) instead of be set again here.

Reply:
@marceloamaral  Actually, I am trying not to import any relative module to config module because it might cause the loop import error. attacher will call config module to get the default config. 
We might also separate all variable definitions to another file and let every module call it.

Reply:
Right!

What about move the constants to the metric file. 
So that we keep all metric names in only one place.

```
const (
	CPUCycleLable       = "cpu_cycles"
	CPUInstructionLabel = "cpu_instr"
	CacheMissLabel      = "cache_miss"
)
```

Reply:
@marceloamaral I just added `pkg/config/types.go`. Please kindly confirm.